.husky/pre-commit

```diff
@@ -0,0 +1 @@
+npx --no-install lerna run --concurrency 1 --stream precommit
```

Comment:
Just curious; why did you add `npx --no-install ...`?

Reply:
Its running Lerna without installing in.
If we don't add npx --no-install it can't find lerna.